pkg/controllermanager/controller/shoot/shoot_control_delete.go

```diff
@@ -415,9 +445,11 @@ func (c *Controller) runDeleteShootFlow(o *operation.Operation) *gardencorev1alp
 	if err := f.Run(flow.Opts{
 		Logger:           o.Logger,
 		ProgressReporter: o.ReportShootProgress,
+		ErrorCleaner:     o.CleanShootTaskError,
+		ErrorContext:     errorContext,
 	}); err != nil {
-		o.Logger.Errorf("Error deleting Shoot %q: %+v", o.Shoot.Info.Name, err)
-		return gardencorev1alpha1helper.LastError(gardencorev1alpha1helper.FormatLastErrDescription(err), gardencorev1alpha1helper.ExtractErrorCodes(flow.Causes(err))...)
+		o.Logger.Errorf("Failed to reconcile Shoot %q: %+v", o.Shoot.Info.Name, err)
```

Comment:
I would prefer `delete` to be used in the error message
```suggestion
		o.Logger.Errorf("Failed to delete Shoot %q: %+v", o.Shoot.Info.Name, err)
```